driver/csiplugin/controllerserver.go

```diff
@@ -635,6 +635,28 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 
 	glog.Infof("volume:[%v] -  spectrum scale volume create params : %v\n", scaleVol.VolName, scaleVol)
 
+	if scaleVol.IsFilesetBased && scaleVol.Compression != "" {
+		glog.Infof("createvolume: compression is enabled: changing volume name")
+		scaleVol.VolName = fmt.Sprintf("%s-COMPRESS%s", scaleVol.VolName, strings.ToUpper(scaleVol.Compression))
```

Comment:
For zfast, We will ask user to create such rule 
WHERE FILESET_NAME LIKE '%COMPRESSZFAST%
and 
For Z, We will ask user to create such rule 
WHERE FILESET_NAME LIKE '%COMPRESSZ%

Does sequence of these rule cause issue ? For example does second rule('%COMPRESSZ%) also compress fileset created for ZFAST algo.